src/BoundaryConditions/fill_halo_regions.jl

```diff
@@ -166,7 +166,8 @@ end
     ntuple(Val(length(bottom_bc))) do n
         Base.@_inline_meta
         @inbounds begin
-            _fill_bottom_halo!(i, j, grid, c[n], bottom_bc[n], loc[n], args...)
+            #_fill_bottom_halo!(i, j, grid, c[n], bottom_bc[n], loc[n], args...)
+            _fill_flux_bottom_halo!(i, j, 1, grid, c[n])
```

Comment:
@simone-silvestri whats this?

Reply:
I don't know, you put it in. Maybe because of #2709?

Reply:
Regardless of who did it, should we revert back to `_fill_bottom_halo!`?

Reply:
Yeah, we have to.